### PR TITLE
fix: add user-agent to kube api calls - job

### PR DESCRIPTION
**Pull request by DeepaliAloori**
https://github.com/razee-io/Razee/issues/142

### CODE REVIEW

File: kubernetes/job/resource.yaml [hunk 1]
```diff
@@ -73,6 +73,11 @@ spec:
         command: ["./bin/razeedeploy-delta", "{{ COMMAND }}", "--namespace={{{ NAMESPACE }}}"]
         args: {{ ARGS_ARRAY }}
         # see README.md for args options. https://github.com/razee-io/razeedeploy-delta/blob/master/README.md
+      env:
+        - name: USER_AGENT_NAME
+          value:  razee-io/razeedeploy-delta
+        - name: USER_AGENT_VERSION
+          value:  "{{{TRAVIS_TAG}}}"
```
Comment (alewitt2) [inline kubernetes/job/resource.yaml:80]:
```suggestion
        env:
          - name: USER_AGENT_NAME
            value:  razee-io/razeedeploy-delta
          - name: USER_AGENT_VERSION
            value:  "{{{TRAVIS_TAG}}}"
        command: ["./bin/razeedeploy-delta", "{{ COMMAND }}", "--namespace={{{ NAMESPACE }}}"]
        args: {{ ARGS_ARRAY }}
        # see README.md for args options. https://github.com/razee-io/razeedeploy-delta/blob/master/README.md
```

**Review by alewitt2 — CHANGES_REQUESTED** [review]
can you also add this change to https://github.com/razee-io/razeedeploy-delta/blob/master/kubernetes/razeedeploy-delta/resource.yaml#L125

**Comment by DeepaliAloori** [comment]
> can you also add this change to https://github.com/razee-io/razeedeploy-delta/blob/master/kubernetes/razeedeploy-delta/resource.yaml#L125

 
Added the changes and opened a PR

File: kubernetes/job/resource.yaml [hunk 2]
```diff
@@ -68,11 +68,16 @@ spec:
         runAsGroup: <%NODE_GROUP_ID%>
         fsGroup: <%NODE_GROUP_ID%>
       containers:
-      - name: "razeedeploy-job{{{ UUID }}}"
+       - name: "razeedeploy-job{{{ UUID }}}"
```
Comment (alewitt2) [inline kubernetes/job/resource.yaml:71]:
```suggestion
      - name: "razeedeploy-job{{{ UUID }}}"
```

File: kubernetes/job/resource.yaml [hunk 3]
```diff
@@ -70,9 +70,14 @@ spec:
       containers:
       - name: "razeedeploy-job{{{ UUID }}}"
         image: "quay.io/razee/razeedeploy-delta:<%TRAVIS_TAG%>"
-        command: ["./bin/razeedeploy-delta", "{{ COMMAND }}", "--namespace={{{ NAMESPACE }}}"]
+       env:
+         - name: USER_AGENT_NAME
+           value: razee.io/razeedeploy-delta
+         - name: USER_AGENT_VERSION
+           value: "{{{TRAVIS_TAG}}}"
+       command: ["./bin/razeedeploy-delta", "{{ COMMAND }}", "--namespace={{{ NAMESPACE }}}"]
```
Comment (alewitt2) [inline kubernetes/job/resource.yaml:78]:
```suggestion
        env:
          - name: USER_AGENT_NAME
            value: razee.io/razeedeploy-delta
          - name: USER_AGENT_VERSION
            value: "{{{TRAVIS_TAG}}}"
        command: ["./bin/razeedeploy-delta", "{{ COMMAND }}", "--namespace={{{ NAMESPACE }}}"]
```

File: kubernetes/job/resource.yaml [hunk 4]
```diff
@@ -70,9 +70,14 @@ spec:
...
+        env:
+          - name: USER_AGENT_NAME
+            value: razee.io/razeedeploy-delta
+          - name: USER_AGENT_VERSION
+            value: "{{{TRAVIS_TAG}}}"
         command: ["./bin/razeedeploy-delta", "{{ COMMAND }}", "--namespace={{{ NAMESPACE }}}"]
         args: {{ ARGS_ARRAY }}
         # see README.md for args options. https://github.com/razee-io/razeedeploy-delta/blob/master/README.md
-      restartPolicy: Never
+        restartPolicy: Never
```
Comment (alewitt2) [inline kubernetes/job/resource.yaml:81]:
```suggestion
      restartPolicy: Never
```

**Review by alewitt2 — DISMISSED** [review]

**Review by alewitt2 — APPROVED** [review]

**Pull request merged**